operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java

```diff
@@ -670,6 +676,37 @@ public NextAction onSuccess(
     return na;
   }
 
+  private static Step getRereadDomainConflictStep(
+      DomainPresenceInfo info, V1ObjectMeta meta, Step next) {
+    return new CallBuilder()
+        .readDomainAsync(
+            meta.getName(),
+            meta.getNamespace(),
+            new ResponseStep<Domain>(next) {
+              @Override
```

Comment:
If you use DefaultResponseStep, you will get much of this behavior automatically. That will improve readability by avoiding needless duplication.